storybook/native/main.js

```diff
@@ -20,7 +20,8 @@ configure(() => {
 
 // add decorators
 const SpaceAround = styled.View`
-  margin: 20px;
+  flex: 1;
+  padding: 10px;
```

Comment:
What was wrong with margin?

Reply:
I was testing for some full-screen background-color and so used padding instead of margin.  It won't break anything.